.github/workflows/ci.yml

```diff
@@ -15,15 +15,19 @@ jobs:
 
     strategy:
       matrix:
-        laravel: ['^9.0']
+        include:
+        - laravel: "9.0"
+          php: "8.0"
+        - laravel: "10.0"
+          php: "8.1"
 
     steps:
       - name: Checkout
         uses: actions/checkout@v2
 
       - name: Install Composer dependencies
         run: |
-          composer require "laravel/framework:${{ matrix.laravel }}" --no-interaction --no-update
+          composer require "laravel/framework:^${{ matrix.laravel }}" --no-interaction --no-update
```

Comment:
Make this the same as in https://github.com/archtechx/tenancy/blob/3.x/.github/workflows/ci.yml. Laravel versions are just `9` and `10` and the `composer require` line has `^${{ matrix.laravel }}.0`

Reply:
https://github.com/archtechx/tenancy/pull/1071/commits/128ff12840dcbed00e4275d100c155a23ef47971